students/seyong/users/views.py

```diff
@@ -1,3 +1,44 @@
-from django.shortcuts import render
+import json, re
 
-# Create your views here.
+from django.http  import JsonResponse, request
```

Comment:
현재 코드에서 사용하고 있지 않은 불필요한 모듈은 지워주세요!
```suggestion
from django.http  import JsonResponse
```